content/en/docs/concepts/policy/api-objects/resource-quotas.md

```diff
@@ -2,7 +2,7 @@
 reviewers:
 - derekwaynecarr
 title: Resource Quotas
-content_type: concept
+content_type: page
```

Comment:
```suggestion
content_type: concept
```
Why did you propose this change? `concept` is the best fit for this front matter value.